conda-env/dev.yml

```diff
@@ -52,4 +49,19 @@ dependencies:
     - matplotlib=3.5.2
     - jupyterlab=3.4.5
     - tbump=6.9.0
+
+requirements:
+    # per https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained
+    run_constrained:
+        # ==================
+        # Deal with existing limitation on osx-arm64
+        # Base
+        # ==================
+        - esmpy=8.2.0 # https://github.com/conda-forge/esmpy-feedstock/issues/55
```

Comment:
This current config ensures that even for [`linux-64` where `esmpy` is available](https://anaconda.org/conda-forge/esmpy/files?version=8.2.0&channel=main) this module is not downloaded and installed - so figuring out the optional dependency will need some more work